src/assets/custom/css/main.scss

```diff
@@ -20,7 +20,7 @@ $vertical-overlap: 100px;
 @import "hero-teaser";
 @import "page-location-anchor";
 @import "section-intro";
-@import "teaser-group";
+@import "teaser-group-card";
```

Comment:
As discussed maybe there's no need to extract that `...card` just yet. It leaves too little for the `...group` to do